### DISCUSSION TITLE
I'd like to show left Navbar allways

### DISCUSSION THREAD

**Discussion by leo-esaki** [thuliteio/doks#1046] [Q&A] [answered]
I'm building an application using @hyas/doks.
On default, we can see left Navbar only in content pages, but I would like make the app shows the Navbar including on landing page always.
How can I implement that?
I need your help urgently.

**Comment by h-enk** [comment, marked as answer]
Please see https://github.com/h-enk/doks/discussions/343

**Comment by leo-esaki** [comment]
Thanks Henk
